src/components/02-components/past-experiences-list/past-experiences-list.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  Box,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Typography,
+} from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import Link from '../../01-elements/link/link';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+
+export type PastExpericiencesListProps = {
+  experienceInfo: { experience: string; status: string }[];
+  ctaLink: string;
+};
+
+export default function PastExpericiencesList({
+  experienceInfo,
+  ctaLink,
+}: PastExpericiencesListProps) {
+  const theme = useTheme();
+
+  // Styles.
+  const contentStyle = {
+    display: 'flex',
+    flexDirection: 'column',
+    flexBasis: '75%',
+  };
```

Comment:
I don't think this is necessary. This should be added only when setting up the component in the Remix app.